subsys/bluetooth/audio/vocs_client.c

```diff
@@ -0,0 +1,668 @@
+/*  Bluetooth VOCS - Volume Offset Control Service - Client */
+
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <zephyr/types.h>
+
+#include <device.h>
+#include <init.h>
+#include <sys/check.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/audio/vocs.h>
+
+#include "vocs_internal.h"
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_VOCS_CLIENT)
+#define LOG_MODULE_NAME bt_vocs_client
+#include "common/log.h"
+
+static struct bt_vocs vocs_insts[CONFIG_BT_MAX_CONN * CONFIG_BT_VOCS_CLIENT_MAX_INSTANCE_COUNT];
+
+static struct bt_vocs *lookup_vocs_by_handle(struct bt_conn *conn, uint16_t handle)
+{
+	__ASSERT(handle != 0, "Handle cannot be 0");
+	__ASSERT(conn, "Conn cannot be NULL");
+
+	for (int i = 0; i < ARRAY_SIZE(vocs_insts); i++) {
+		if (vocs_insts[i].cli.conn == conn &&
+		    vocs_insts[i].cli.active &&
+		    vocs_insts[i].cli.start_handle <= handle &&
+		    vocs_insts[i].cli.end_handle >= handle) {
+			return &vocs_insts[i];
+		}
+	}
```

Comment:
Empty line after `}` (unless the next line is also `}`. I think I saw at least one other place in the PR with the same issue. (minor thing, I know, but something we've tried to be consistent with within the Bluetooth subsystem)

Reply:
Agreed. This, imo, is however something a linter should handle for us, and not something you, as a reviewer, should need to deal with. Whatever linting we have that require empty line after a variable declaration: Can't that do this as well?

Reply:
Probably, but I'm not sure this rule is followed much outside of the Bluetooth subsystem, so that might make it hard to enforce it on others. Linux checkpatch has support for subsystem-specific quirks, but I don't think we want to go that route with Zephyr. There are all sorts of small personal preferences which I feel like they help readability, but probably not worth codifying anywhere. Another which I've stopped pointing out about is the "reverse x-mas tree" rule for variable declarations where the shortest lines ones come last 😄 

Even for this curly-brace rule, I'm not sure it applies in all cases, since the most important thing about empty lines is to group things which logically belong together into a single block. There could be cases where the line immediately following a block is so strongly related that adding the empty line detracts from easy understanding of the code. And that again means that automating this isn't really possible.

Reply:
I agree :) My personal opinion is that any code style things (such as empty lines after variable declaration, or code blocks) should be handled by a linter. A lot of PR reviews become longer due to such comments (although it's generally good that they are there!), which could have been fixed by the uploader before assigning reviewers :) 

But as you also say, it's hard to apply such rules across the entirety of Zephyr as there are so many code styles applied in different subsystems.